test/build/plugin-sass/plugin-sass.test.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const path = require('path');
+const {setupBuildTest} = require('../../test-utils');
+
+const cwd = path.join(__dirname, 'build');
+
+describe('@snowpack/plugin-sass', () => {
+  beforeAll(() => {
+    setupBuildTest(__dirname);
+  });
+
+  it('ignores partials', () => {
+    const buildContents = fs.readdirSync(path.join(cwd, '_dist_'));
+    expect(buildContents).toEqual(['index.css']); // assert index.css AND ONLY index.css is output (ignore _partial.scss)
```

Comment:
This test has 2 files: `index.scss` and `_partial.scss`. The test asserts that only the former is left in the build.